ros2_socketcan_msgs/msg/Frame.msg

```diff
@@ -0,0 +1,8 @@
+std_msgs/Header header
+uint32 id
+bool is_rtr
+bool is_extended
+bool is_error
+bool is_fd_frame
+uint8 dlc
+uint8[64] data
```

Comment:
Because it more closely matches the original [`can_msgs/Frame`](https://github.com/ros-industrial/ros_canopen/blob/e10f4905c6b773f8f31f99b900665a69a6a7678a/can_msgs/msg/Frame.msg#L7). I'm fine with `uint8[<=64] but an unbounded array seems unnecessary.

Reply:
I've followed these discussions for quite some time now because we also need CAN FD support for taking the radar sensors of our research vehicle into full operation. If necessary, I can offer my help with the implementation.

From my point of view, changing this to uint8[<=64]  (and option 3 in issue #21) seems like a good idea, because this reflects the structure of CAN FD frames. Besides that, the message definition looks good to me and I hope that this gets merged soon.